ethexe/service/src/tests.rs

```diff
@@ -1589,20 +1584,29 @@ mod utils {
             self.apply_until_block_event_with_header(|e, _h| f(e)).await
         }
 
+        // TODO: consider to iterates also thru skipped blocks
```

Comment:
whats skipped blocks in this context? you mean for blocks that alloy didn't share headers of due to brut force bump of bn?

Reply:
Yes, I mean blocks which observer skips because they were skipped by alloy herder stream. This is possible on reorgs for example. Currently it's useless for us to look on these blocks, but maybe in future we would have a test where skipped blocks contains sensetive for test events.